src/backend/aspen/workflows/nextstrain_run/autorun_scheduled.py

```diff
@@ -0,0 +1,92 @@
+import datetime
+import json
+import os
+import re
+from copy import deepcopy
+from typing import Dict, MutableSequence
+
+import click
+import sqlalchemy as sa
+from boto3 import Session
+from botocore.exceptions import ClientError
+
+from aspen.config.config import Config
+from aspen.database.connection import (
+    get_db_uri,
+    init_db,
+    session_scope,
+    SqlAlchemyInterface,
+)
+from aspen.database.models import Group
+
+SCHEDULED_TREE_TYPE = "OVERVIEW"
+
+TEMPLATE_ARGS = {"filter_start_date": "12 weeks ago", "filter_end_date": "now"}
+
+
+def launch_scheduled_run(aws_client, sfn_params: Dict, group: Group):
+    start_datetime = datetime.datetime.now()
+
+    sfn_params["Input"]["Run"] |= {
+        "group_name": group.name,
+        "s3_filestem": f"{group.prefix}/scheduled",
+        "template_args": json.dumps(TEMPLATE_ARGS),
+        "tree_type": SCHEDULED_TREE_TYPE,
+    }
+    sfn_params[
```

Comment:
I think a comment about why it's important to make output_prefix unique here would be good!